utils/cLog.h

```diff
@@ -65,7 +68,8 @@ class cLogger {
       cLogger(const cLogger&) = delete;
       cLogger(cLogger&&) = delete;
 
-      std::FILE* m_file;
+      //std::FILE* m_file;
```

Comment:
Let's just remove the commented out lines.

Reply:
Done